data/multiplayer/scenarios/4p_A_New_Land.cfg

```diff
@@ -497,6 +497,12 @@ For experienced players, it’s recommended to lower the starting gold to 75."
             image=terrain/village/cave-tile.png
             message= _ "You destroy the mushroom mine. The northwest undead now have 10 less income."
         [/message]
+        [progress_achievement]
+            content_for=wesnoth_MP_survivals
+            id=Wesnoth_MP_ANL_mines
+            amount=1
+            limit=1
```

Comment:
From my understanding the limit needs to be a variable, which is keeping track of how many were destroyed.

Reply:
Kind of confused here, since this event macro is called 4 times in the scenario so I assumed it was event-specific and not scenario specific

Reply:
What I mean is that it should look something like this:
```
[set_variable]
    name=mines_destroyed
    add=1
[/set_variable]
[progress_achievement]
    # same stuff here
    amount=1
    limit=$mines_destroyed
[/progress_achievement]
```

My understanding is that it clamps to at most the limit, so with a limit of 1, it will only ever record one mine destroyed.

Reply:
That's correct.